it/src/main/java/org/corfudb/universe/universe/vm/VmUniverseParams.java

```diff
@@ -0,0 +1,81 @@
+package org.corfudb.universe.universe.vm;
+
+import lombok.Builder;
+import lombok.Builder.Default;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.ToString;
+import org.corfudb.universe.universe.Universe;
+import org.corfudb.universe.universe.UniverseParams;
+
+import java.time.Duration;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+/**
+ * Represents the parameters for constructing a VM {@link Universe}.
+ */
+@Getter
+@ToString(callSuper = true)
+public class VmUniverseParams extends UniverseParams {
+    @NonNull
+    private final String vSphereUrl;
+    @NonNull
+    private final String vSphereUsername;
+    @NonNull
+    private final String vSpherePassword;
+    @NonNull
+    private final String vSphereHost;
+    @NonNull
+    private final String templateVMName;
+    @NonNull
+    private final String vmUserName;
+    @NonNull
+    private final String vmPassword;
+    @NonNull
+    private final ConcurrentMap<String, String> vmIpAddresses;
+    @NonNull
+    @Default
+    private final String domainName = "eng.vmware.com";
+    @NonNull
+    @Default
+    private final String timeZone = "America/Los_Angeles";
+    @NonNull
+    @Default
+    private final String[] dnsServers = new String[]{"10.172.40.1", "10.172.40.2"};
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make sure using this hardcoded IP address is safe here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1313)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make sure using this hardcoded IP address is safe here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1313)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)